osu.Game/Overlays/Chat/Tabs/ChannelTabItem.cs

```diff
@@ -123,6 +115,8 @@ public ChannelTabItem(Channel value)
 
         protected virtual bool ShowCloseOnHover => true;
 
+        protected virtual bool IsBoldWhenActive => true;
```

Comment:
field name seems a bit off (as if it was a check), I think `BoldOnActive` would be better

Reply:
Disagree on this one. Is consistent with `IsSwitchable` and `IsRemovable`

Reply:
what about `ShowCloseOnHover`